source/_includes/common-tasks/backups.md

```diff
@@ -57,19 +67,60 @@ There are multiple ways to store the backup on another device:
-
-When the restore is complete, Home Assistant will restart to apply the new settings. You will lose the connection to the UI and it will return once the restart is completed.
+There are two ways to use a backup:
+
+- On your current system to recover your settings.
+- During onboarding, to migrate your setup to a new device or a to device on which you performed a factory reset and a new installation.
+
+#### Restoring a backup during onboarding
+
+You can use a backup during the onboarding process to restore your configuration.
+
+**Migration**: This procedure also works if you want to migrate from one device to another. In that case, use the backup of the old device on the new device. The target device can be a different device type. For example, you can migrate from a Raspberry&nbsp;Pi to another smart home hub.
+
+##### Prerequisites
+
+1. The login credentials of the device from which you made the backup.
+2. If you migrate the installation to a new device, make sure the new device has more storage capacity than the existing device.
+   - Before migrating, on the old system, check how much storage you used.
+     - Go to **{% my storage title="Settings > System > Storage" %}**, and under **Disk metrics**, hover over the **Used space** bar.
+     - The tooltip shows your disk size.
+     - The target device must have more free space than the source device.
+        - If your target device is a Home Assistant Yellow, note that it is the size of the eMMC that is relevant.
+        - The restore process mainly uses the eMMC, not the NVMe.
+3. If you are migrating to a new device:
+   - You do not need to transfer the backup to a USB or SD card to bring it to your smart home hub.
+   - You will be able to access the explorer of your PC.
+4. This procedure assumes you have already completed the [installation](/installation/) procedure on your target device and are now viewing the welcome screen as part of the [onboarding](/getting-started/onboarding/).
+
+Note: Restoring from a backup will clear all data you had on this device.
+
+##### To restore a backup during onboarding
+
+1. If you are migrating to a new device and you had controllers or radios connected (such as a Z-Wave stick or SkyConnect):
+   - make sure to plug them into the new device.
+2. After Home Assistant has been installed, on the welcome screen, select **Restore from backup**.
+   - Then, select **Upload backup**.
+   - The file explorer opens on the device on which you are viewing the Home Assistant User interface.
+   - You can access any connected network drive from there.
+3. Select the backup file, then, in the dialog, select **Full backup** and **Restore**.
+   - The restore may take a while, depending on the amount of data.
+   - To see if the restore is complete, reload the page from time to time.
```

Comment:
Should we add a note here about SSL?
If your previous installation had certificates enabled directly for the `http` integration, when the restore is complete it will no longer respond to `http://` requests, and the user would need to use `https://` (added `s`) instead